pom.xml

```diff
@@ -181,6 +181,7 @@
                     <artifactSet>
                         <includes>
                             <include>org.apache.jackrabbit:*</include>
+                            <include>com.google.guava:*</include>
```

Comment:
i think we have to relocate it as well to really make sure there are no classpath issues?

Reply:
oh - relocating does not work because it's references not from our code here but from jackrabbit oak itself.
does it then help at all including it without relocating?

Reply:
Relocation does work (as long as the embedded Oak version is used). I therefore relocated in https://github.com/apache/sling-org-apache-sling-testing-sling-mock-oak/pull/5/commits/bdf07b00070b71c5eb2d3a23aa7302300df173ab. That way the order of the class path does not matter.